net/syncmanager/syncmanager.go

```diff
@@ -721,7 +721,10 @@ func (sm *SyncManager) syncPoints(peer *peer.Peer) (pindexWalk, pindexBestKnownB
 // list of blocks to be downloaded based on the current known headers.
 // Download blocks via several peers parallel
 func (sm *SyncManager) fetchHeaderBlocks(peer *peer.Peer) {
-	//log.Debug("now %d requestedBlocks", len(sm.requestedBlocks))
+	reqNum := len(sm.requestedBlocks)
+	if 0 != reqNum {
```

Comment:
if reqNum := len(sm.requestedBlocks);  reqNum != 0  {..}
better inlined like this